package/MDAnalysis/analysis/rms.py

```diff
@@ -569,6 +628,15 @@ def save(self, filename=None):
 class RMSF(AnalysisBase):
     r"""Calculate RMSF of given atoms across a trajectory.
 
+    .. Note::
```

Comment:
please use numpy notes. Also here a good idea is to reference alignment and mention that it has the `in_memory` option for smaller trajectories.